### PR TITLE
Add dev deploy to pypi on merge back

### PR DESCRIPTION
**Pull request by dabacon**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -78,3 +78,18 @@ matrix:
     - pip install -r requirements.txt
     - pip install -r cirq/contrib/contrib-requirements.txt
     script: check/doctest -q
+  - stage: deploy
+    os: linux
+    env: NAME=dev-release
+    python: '3.7'
+    script: export CIRQ_DEV_VERSION=`dev_tools/packaging/.sh`
```
Comment (cduck) [inline .travis.yml]: Typo.

Comment (dabacon) [reply .travis.yml]: finds cursor on spot.  ctrl-z fix.  doh

File: .travis.yml [hunk 2]
```diff
@@ -78,3 +78,18 @@ matrix:
     - pip install -r requirements.txt
     - pip install -r cirq/contrib/contrib-requirements.txt
     script: check/doctest -q
+  - stage: deploy
+    os: linux
+    env: NAME=dev-release
+    python: '3.7'
+    script: export CIRQ_DEV_VERSION=`dev_tools/packaging/generate-dev-version-id.sh`
```
Comment (Strilanc) [inline .travis.yml]: Nit Nit: I like `$(...)` instead of backticks.

Comment (dabacon) [reply .travis.yml]: changed

**Pull request merged**